### PR TITLE
Refactor component and delete unused files

### PR DESCRIPTION
**Pull request by amadzuki**
Refactor component and delete unused files

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -9,7 +9,7 @@ import MangaCollection from './pages/manga'
 import Register from './pages/register'
 import Login from './pages/login'
 
-import ScrollToTop from './utils/ScrollToTop'
+import ScrollToTop from './components/ScrollToTop'
```
Comment (mhaidarhanif) [inline src/App.js:12]: Yes, ScrollToTop should be a component, not a `utils`.

**Review by mhaidarhanif — CHANGES_REQUESTED** [review]
I think the changes are already good.

**Comment by amadzuki** [comment]
Ok, is it alright to merge now?

**Comment by mhaidarhanif** [comment]
Okay, approved.

**Review by mhaidarhanif — APPROVED** [review]
All done!

**Pull request merged**